source/includes/_content_access_management.md

```diff
@@ -0,0 +1,283 @@
+# Content Access
+
+## Access Basics
+
+For each content item your organization uploads you will need coresponding access records. Access records define who can use the item for their ad campaigns and whether or not they have the ability to edit the item after it is in the system. Content Items have a one to many relationship to access records, although it is completely valid to have a content item only accessible by a single user.
+
+Access to a given content item can be defined at the following levels:
+- User based
+- Group Based
+- Group and Role based
+
+It's important to note that you you can create multiple access records tying content items to multiple 
```

Comment:
s/you you/you

Reply:
also this line ends abruptly.

Reply:
Yep! Looks like I stopped mid sentence :-1: